Dockerfile

```diff
@@ -25,11 +25,43 @@ RUN <<EOF
   adduser --quiet --system --group --disabled-password --home /var/lib/clamav --no-create-home --uid 200 clamav
 EOF
 
-COPY target/scripts/build/* /build/
+COPY target/scripts/build/packages.sh /build/
 COPY target/scripts/helpers/log.sh /usr/local/bin/helpers/log.sh
 
 RUN /bin/bash /build/packages.sh && rm -r /build
 
+
+
+# -----------------------------------------------
+# --- Compile deb packages ----------------------
+# -----------------------------------------------
+
+FROM stage-base AS stage-compile
```

Comment:
May not be worth it, but presently this is dependent on `stage-base` installing packages, whereas that's all the prior stage is really doing differently, it could build in parallel if it wasn't dependent upon that (_may affect our CI build cache size though, so perhaps the slower build approach here is acceptable_).